libraries/RC_Channel/RC_Channel.h

```diff
@@ -199,7 +199,7 @@ class RC_Channel {
         TRAINING            = 98, // mode training
         AUTO_RTL =            99, // AUTO RTL via DO_LAND_START
 
-        // entries from 100 onwards are expected to be developer
+        // entries from 100-150  are expected to be developer
         // options used for testing
         KILL_IMU1 =          100, // disable first IMU (for IMU failure testing)
```

Comment:
Nothing to do with this PR, but I wonder if we should move these, there now well within typo range now. Unfortunately I don't really see anyway to reclaim those numbers again.